Migrate SQL docs (#521)
redpanda-data/cloud-docs: modules/reference/pages/sql/sql-functions/timestamp-functions/timestamp-millis.adoc

= TIMESTAMP_MILLIS
:description: The TIMESTAMP_MILLIS() function converts a given UNIX timestamp value in milliseconds since 1970-01-01 00:00:00 UTC into a timestamp.
:page-topic-type: reference

The `TIMESTAMP_MILLIS()` function converts a given UNIX timestamp value in milliseconds since 1970-01-01 00:00:00 UTC into a timestamp. Its syntax is as follows:

[source,sql]
----
SELECT TIMESTAMP_MILLIS(BIGINT)
----

Its input type is a `BIGINT` expression which represents a UNIX timestamp in milliseconds and the return data type is a timestamp.

== Examples

=== Basic `TIMESTAMP_MILLIS()` function

The below example uses the `TIMESTAMP_MILLIS()` function to convert a given UNIX timestamp in milliseconds into a timestamp without a timezone.

[source,sql]
----
SELECT TIMESTAMP_MILLIS(1671975000000) AS timestamp_millisvalues;
----

Output:

[source,sql]
----
+-----------------------------+
| timestamp_millisvalues      |
+-----------------------------+
| 2022-12-25 13:30:00         |
+-----------------------------+
----

=== `TIMESTAMP_MILLIS()` function using columns

> Suppose you have a table named **unix_example** with the following UNIX time values in milliseconds in the *unix_timestamp* column:

Reviewer: ```suggestion
Suppose a table named **unix_example** has the following UNIX time values in milliseconds in the *unix_timestamp* column:
```

[source,sql]
----
CREATE TABLE unix_example (
  unix_timestamp long
);

INSERT INTO unix_example VALUES
('171472000000'),
('1671975000000'),
('153276000000');
----

[source,sql]
----
SELECT * FROM unix_example;
----

> This query shows the following table:

Reviewer: ```suggestion
This query shows the table:
```
